monitoring/defining-monitors.mdx

```diff
@@ -0,0 +1,93 @@
+---
+title: "Defining Monitors"
+description: "Learn how to create and configure monitors to evaluate your LLM outputs"
+---
+
+Monitors in Traceloop allow you to continuously evaluate your LLM outputs in real time. This guide walks you through the process of creating and configuring monitors for your specific use cases.
+
+## Creating a Monitor
+
+To create a monitor, you need to define:
```

Comment:
Worth mentioning again - you need to connect the SDK and send traces, preferably also set up decorators.